skimage/filters/thresholding.py

```diff
@@ -727,14 +727,10 @@ def threshold_triangle(image, nbins=256):
     width /= norm
 
     # Maximize the length.
-    d = peak_height * arg_low_level - width * hist[arg_low_level]
```

Comment:
Then I would suggest adding ImageJ as reference and adding a comment with something like 'the ImageJ implementation includes an additional constant when calculating the length, but here we omit it as it does not affect the location of the minimum'.

Also, I've attached my understanding of the trigonometry. 
![img_20160628_085735](https://cloud.githubusercontent.com/assets/1189568/16423000/32f43f74-3d0f-11e6-8801-50c3b5008aec.jpg)